src/widgets/Header.m.css

```diff
@@ -0,0 +1,209 @@
+.root {
+	--toggle-button-height: 20px;
+	--toggle-button-width: 26px;
+	--header-background-color: #212329;
+	--header-link-color: #999;
+	--shadow-color: rgba(0, 0, 0, 0.5);
+	--toggle-button-color: #d3d3d3;
+	--link-color: #fff;
+	position: fixed;
+	width: 100%;
+	background: var(--header-background-color);
+	display: flex;
+	justify-content: flex-start;
+	align-content: flex-start;
+	align-items: center;
+}
+
+.menu {
+}
+
+.menuList {
+	list-style: none;
+	display: flex;
+	padding: 0;
+	height: auto;
+	width: auto;
+	background: none;
+}
+
+.menuItem {
+	text-align: right;
+	margin: 10px;
+}
+
+.left {
+	display: flex;
+	align-items: center;
+	flex: 1 0 auto;
+}
+
+.link {
+	font-size: 0.9rem;
+	min-width: 140px;
+	text-align: center;
+	text-decoration: none;
+	color: var(--link-color);
+	box-sizing: border-box;
+	color: var(--header-link-color);
+	text-transform: uppercase;
+	transition: 0.1s ease-in-out;
+	transition-property: color, background-color;
+}
+
+.homeLink {
+}
+
+.selected {
+}
+
+.logoContainer {
+}
+
+.logo {
+	height: 48px;
+	width: 110px;
+}
+
+.backdrop {
+}
+
+.expanded .backdrop {
+	position: fixed;
+	display: block;
+	content: '';
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 997;
```

Comment:
Is there a specific reason we use `997`?

Reply:
I wanted these to be high values to ensure they would float over any kind of page content that might be on the site. My methods might be outdated, though. Is there a better way to do this?